### PR TITLE
Courses: add course admin

### PR DESCRIPTION
**Pull request by rbngzlv**
image
image

### CODE REVIEW

**Review by victorol1 — CHANGES_REQUESTED** [review]
Hi @rbngzlv, LGTM, but we have to fix some bugs before merging, the first one is related to the `InviteUserAgain` command as stated in a comment, but there are a few more.

**1.** Department admins can't see courses in the BO even if they have the same area.
**2.** When using a new user invited as a department admin going to the BO results in crash from AdminUsers query not being implemented yet for ResourceBanks. Should be solved after implementing the same on ResourceBanks

We should try to close this task today, I'll re-review later today if needed.

**Comment by rbngzlv** [comment]
Hi @victorol1,

As we have commented in our call, this PR is unrelated with the department admin module (we'll work on this in the next days).

And about the sign in "bug": I can reproduce it when having a pending invitation to be an administrator of either, assembly or course. So, as we thought, it is not related to this PR, but Decidim or the department admin module, because the message _"You have a pending invitation, accept it to finish creating your account."_ is not related with the pending invitation to be a course/assembly admin, but the email received to confirm the account creation, so seems that the users invited are marked as "new users".

In order to not block this PR we are going to investigate this as a possible bug in the next days.

/cc @tramuntanal

**Comment by victorol1** [comment]
Alright @rbngzlv, I'm merging since the error is not related to this PR.

**Pull request merged**
